specification/synapse/data-plane/Microsoft.Synapse/preview/2020-01-01-preview/integrationRuntimes.json

```diff
@@ -0,0 +1,127 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01-preview",
+    "title": "SynapseClient"
```

Comment:
>SynapseClient [](start = 14, length = 13)

Let's name this as `IntegrationRuntimeClient`

Reply:
Thanks, resolved it.